tests/nnstreamer_filter_custom/runTest.sh

```diff
@@ -91,4 +91,15 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/
 python checkScaledTensor.py testcase14.direct.log 640 480 testcase14.scaled.log 1920 1080 3
 casereport 14 $? "Golden test comparison"
 
+# Test average using OpenCV (15)
+# cutom version
+PATH_TO_MODEL_A="../../build/nnstreamer_example/custom_example_average/libnnstreamer_customfilter_average.so"
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=640,height=480,framerate=0/1 ! videoconvert ! video/x-raw, format=RGB ! tensor_converter ! tensor_filter framework=\"custom\" model=\"${PATH_TO_MODEL_A}\" ! filesink location=\"testcase15.average.log\" sync=true" 15
+
+# OpenCV version
+PATH_TO_MODEL_A="../../build/nnstreamer_example/custom_example_opencv/libnnstreamer_customfilter_opencv_average.so"
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=640,height=480,framerate=0/1 ! videoconvert ! video/x-raw, format=RGB ! tensor_converter ! tensor_filter framework=\"custom\" model=\"${PATH_TO_MODEL_A}\" ! filesink location=\"testcase15.opencv.average.log\" sync=true" 15
+
+compareAll testcase15.opencv.average.log testcase15.average.log 15
```

Comment:
The case number (the last argument) would be better if it's 16 or 15-2.

Reply:
Never mind. It's ok with 15.

However, if #729 gets merged before this, you will need to rebase and change a bit anyway.